### ISSUE TITLE
Include modules in firebase object

### ISSUE THREAD

**Issue by mrshll** [prescottprue/redux-firebasev3#18]
In `connect.js`, when we create the firebase connect component, should we expose the storage, auth, and database? Are we keeping those private because we want things to always flow through redux or because they did not exist in v2?

```
const {ref, helpers, storage, database, auth} = firebase
this.firebase = {ref, storage, database, auth, ...helpers}
```

**Comment by prescottprue** [comment]
Correct, they are not exposed so that everything flows through redux.

More then willing to expose them, especially if you have a specific use case for when you would not want to go through redux.

**Comment by mrshll** [comment]
If the import is renamed, as in #17, this isn't as big an issue. Main things were being able to access storage and other firebase methods that were not exposed in `connect` more easily.

**Comment by prescottprue** [comment]
This is included in #22 and the release of [`v0.1.3`](https://github.com/prescottprue/redux-firebasev3/releases/tag/v0.1.3). Let me know what you think.

We can keep this open until we make sure it satisfies your need.

**Comment by prescottprue** [comment]
@mmoutenot Just a heads up, this is also in [`v0.1.4`](https://github.com/prescottprue/redux-firebasev3/releases/tag/v0.1.4). Hope it is working for you.
